cmdb/src/main/java/com/alibaba/nacos/cmdb/memory/CmdbProvider.java

```diff
@@ -117,11 +117,9 @@ public void init() throws NacosException {
         initCmdbService();
         load();
         
-        UtilsAndCommons.GLOBAL_EXECUTOR.schedule(new CmdbDumpTask(), switches.getDumpTaskInterval(), TimeUnit.SECONDS);
-        UtilsAndCommons.GLOBAL_EXECUTOR
-                .schedule(new CmdbLabelTask(), switches.getLabelTaskInterval(), TimeUnit.SECONDS);
-        UtilsAndCommons.GLOBAL_EXECUTOR
-                .schedule(new CmdbEventTask(), switches.getEventTaskInterval(), TimeUnit.SECONDS);
+        UtilsAndCommons.scheduleCmdbTask(new CmdbDumpTask(), switches.getDumpTaskInterval(), TimeUnit.SECONDS);
```

Comment:
Should we create a new class CmdbExecutors like naming and config? Don't put them into UtilsAndCommons